src/pizzeria/Order.java

```diff
@@ -0,0 +1,84 @@
+package pizzeria;
+
+import java.time.LocalTime;
+import java.util.*;
+
+public class Order {
+    private final int orderNumber;
```

Comment:
You have to create Pizza class and put their fields and methods inside Pizza class.

for example pizzaName, quantityOfPizza, pizzaType, ingredients, etc.

Reply:
but should quantityOfPizza be in Pizza class, when it refers to order?

Reply:
For the calculation of the order's pizzas' quantity you have to create OrderItem class as you already discussed in your last lesson.

Reply:
And also, please create a Customer class and move their related fields and logic there.